### ISSUE TITLE
Allow setting enum values by name

### ISSUE THREAD

**Issue by matze** [ufo-kit/uca-net#8]
Right now, only numerical values can be set.

**Comment by matze** [comment]
Should work now with latest commit and libuca.

**Comment by matze** [comment]
@tfarago and @MarcusZuber: could you please tell me if f8efbe70906 fixed your issues? This is what is possible in concert at the moment (assuming `ucad` serves a mock camera):

```python
camera = Camera('net', {'host': 'local:8989'})
camera.test_enum = camera.uca.enum_values.test_enum.UCA_MOCK_CAMERA_TEST_ENUM_BAR
```

You problem was that this is not working or that it is too long?

**Comment by tfarago** [comment]
Not really, on the server side:
`camera.timestamp_mode = camera.uca.enum_values.timestamp_mode.ASCII` and on the client side I need `camera.timestamp_mode = camera.uca.enum_values.timestamp_mode.UCA_PCO_CAMERA_TIMESTAMP_ASCII`. BTW, the commit you are referencing dates back to December, is that correct?

**Comment by tfarago** [comment]
So I can set the value, but the name changes depending if I use the camera directly or via `uca-net`, which messes up all my `concert` sessions.

**Comment by matze** [comment]
> BTW, the commit you are referencing dates back to December, is that correct?

Yes.

> So I can set the value, but the name changes

So, then tell me which one you prefer …

**Comment by tfarago** [comment]
The shorter version with just the constant name please.

**Comment by matze** [comment]
Alright, the commit that closed #10 should fix this issue.

**Comment by MarcusZuber** [comment]
@tfarago Did you test this?
